spec/controllers/social_networking/nudges_controller_spec.rb

```diff
@@ -84,5 +84,14 @@ module SocialNetworking
         end
       end
     end
+
+    describe "private methods" do
```

Comment:
Weird, I swear I did a Ctrl-F! However, it's not a great idea to test private methods: you should be able to change the method name and/or refactor it without features or specs breaking. Usually if you want to test a private method the approach would be to create a new class.